util/dasharo/tag-with-changelog

```diff
@@ -410,6 +422,26 @@ for module in $(git submodule --quiet foreach pwd); do
 	get_log_submodule "${!old_version}" "${!new_version}" "${path}"
 done
 
+echo "##############" >> "$LOGFILE"
+echo "#### EDK2 ####" >> "$LOGFILE"
+echo "##############" >> "$LOGFILE"
+# Hardcoded, probably won't ever change
+URL="https://github.com/Dasharo/edk2"
+# This is the first digest from common Kconfig option. While it could be pointed
+# to by previous release (in which case the changelog would be complete), it
+# isn't trivial to differentiate between such case and use as placeholder value.
+PREHISTORY="f0459dff1425c0c02d0736de5737eb23a1b671ec"
+
+if [ -z $OLD_EDK2_GIT_VERSION ]; then
+	echo "# Old EDK2 version couldn't be obtained, changelog may be incomplete!" >> "$LOGFILE"
+	echo "# Commits older than ${PREHISTORY:0:10} won't be shown" >> "$LOGFILE"
+	OLD_EDK2_GIT_VERSION="$PREHISTORY"
+fi
+
+git clone "$URL" || version_ctrl_c
```

Comment:
This should be almost as good as reusing that repository when it exists:
```suggestion
git clone --reference-if-able payloads/external/edk2/workspace/Dasharo/ --bare "$URL" edk2 || version_ctrl_c
```
If that path doesn't exist, the repo gets cloned as usual.  `edk2` at the end is needed because `--bare` results in `edk2.git` name being used by default.

Reply:
https://github.com/Dasharo/coreboot/compare/6116e4b5e35d9173a29ac3fa1e4932ff8b6e60f0..0b42ba7fcb2c81fe05999cd29fd4d93c3cce8555#diff-0ecf7672503aa0c7446284c3c0b00569b847900b9e4e0db39822bf42fb7ba62fR441

It prints a warning when the directory doesn't exist, but since it prints download progress anyway, it may serve as an information as to why the download is required in the first place.